suite-native/staking/src/ethereum/ethereumSelectors.ts

```diff
@@ -0,0 +1,101 @@
+import { NetworkSymbol } from '@suite-common/wallet-config';
```

Comment:
What about naming this file ethereumStakingSelectors.ts for better context?

Reply:
yea, sounds nice